stable/kubewatch/README.md

```diff
@@ -37,32 +37,40 @@ The command removes all the Kubernetes components associated with the chart and
 
 The following table lists the configurable parameters of the kubewatch chart and their default values.
```

Comment:
can you also update the link at the top of the readme to point to the bitnami-labs repo, instead of skipbox

Reply:
yes, I missed that.